tests/protocol/test_extensions_goal_state_from_vm_settings.py

```diff
@@ -19,7 +18,7 @@ def assert_property(name, value):
 
             assert_property("activity_id", "a33f6f53-43d6-4625-b322-1a39651a00c9")
             assert_property("correlation_id", "9a47a2a2-e740-4bfc-b11b-4f2f7cfe7d2e")
-            assert_property("created_on_timestamp", "2021-11-16T13:22:50.620522Z")
+            assert_property("created_on_timestamp", "2021-11-16T13:22:50.620529Z")
```

Comment:
updated the timestamp on a few data files to make them the most recent goal state